hack/fetch_ext_bins.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
```

Comment:
I see CAPA doing the same thing. Is this the only way to setup the test binaries? We also have the `/hack/tools` directory we use to go build binaries into

Reply:
The only concern here would be how resource intensive it is to `go build` these binaries locally. And if it takes more time than downloading the binaries, then sticking to this option would make more sense.
I can start an exploratory issue and continue some work there and let  this one merge for now. So that we can start accepting new PRs for more controller tests.

Reply:
thanks Sagar. Was more curious than anything.